packages/calcite-components/src/components/filter/filter.tsx

```diff
@@ -181,6 +182,20 @@ export class Filter
   //
   // --------------------------------------------------------------------------
 
+  /**
+   * Sets focus on the component.
```

Comment:
Doc description does not match function.

On a related note, I think we should add a little snippet regarding the need for this method since filtering is delayed and asynchronous.